tests/unittests/CoreGraphics.drawing/CGContextDrawing_GradientTests.cpp

```diff
@@ -37,15 +37,27 @@ static void _drawShortLinearGradientWithOptions(CGContextRef context, CGRect bou
     CGFloat locations[2] = { 0, 1 };
     CGFloat components[8] = { 0.0, 1, 0.0, 1.0, 1.0, 0, 0, 1.0 };
 
-    CGColorSpaceRef colorspace = CGColorSpaceCreateDeviceRGB();
-    CGGradientRef gradient = CGGradientCreateWithColorComponents(colorspace, components, locations, 2);
-
     CGRect borderRect = CGRectInset(bounds, 30, 50);
 
-    CGContextDrawLinearGradient(context, gradient, borderRect.origin, CGPointMake(borderRect.size.width, borderRect.size.height), option);
+    _drawLinearGradient(context,
+                        borderRect.origin,
+                        CGPointMake(borderRect.size.width, borderRect.size.height),
+                        components,
+                        locations,
+                        _countof(locations),
```

Comment:
use extent instead of _countof which is MS only and shouldn't work on reference platform.